### DISCUSSION TITLE
unable to connect to Hive MQTT

### DISCUSSION THREAD

**Discussion by Nairodut** [jomjol/AI-on-the-edge-device#2619] [General]
Can someone please help me? 
![Screenshot 2023-09-21 123223](https://github.com/jomjol/AI-on-the-edge-device/assets/145662115/3c044a75-e79b-4dda-b08c-c7f0fa1c93d4)
![Screenshot 2023-09-21 122817](https://github.com/jomjol/AI-on-the-edge-device/assets/145662115/083ecc94-4a47-4103-8acf-a3d330eebf7e)

**Comment by caco3** [comment]
Ist die URI korrekt, sieht nicht wie eine Domain oder IP aus!
Und das Passwort ist nicht so, oder?

Kommst Du mit den gleichen Zugangsdaten und einem anderen MQTT client (z.b. http://mqtt-explorer.com/) auf den MQTT Broker?
